pyndulum/pendulum_equations.py

```diff
@@ -92,3 +92,6 @@ def bpm(len: float) -> float:
         pendulum frequency in beats per minute [1 / min]
     """
     return 60.0 / get_period(len)
+
+def inv(p):
```

Comment:
I'd like a better name for this function that is more self-explanatory.

Reply:
Good idea. On it!

Reply:
Done